front/src/schemas.ts

```diff
@@ -334,6 +334,9 @@ export const SpectrogramParametersSchema = z
     channel: z.coerce.number().nonnegative().int().default(0),
     pcen: z.boolean().default(false),
     cmap: z.string().default(DEFAULT_CMAP),
+    gamma: z.coerce.number().positive().default(2),
+    low_signal: z.coerce.number().nonnegative().default(0.3),
+    high_signal: z.coerce.number().nonnegative().default(0.5),
```

Comment:
Would be better to import the default values from the constants.ts module for easier maintenance.